lib/pluginmanager/package.rb

```diff
@@ -0,0 +1,32 @@
+class LogStash::PluginManager::Package < LogStash::PluginManager::Command
+
+  option "--tgz", :flag, "compress package as a tar.gz file", :default => false
+  option "--zip", :flag, "compress package as a zip file", :default => false
+  option "--clean", :flag, "clean up the generated dump of plugins", :default => false
+
+  def execute
+    signal_error("File #{LogStash::Environment::GEMFILE_PATH} does not exist or is not writable, aborting") unless File.writable?(LogStash::Environment::GEMFILE_PATH)
```

Comment:
```GEMFILE_PATH``` points to the ```Gemfile```, I just used the regular checkup existence in the other plugin manager requests. Makes sense? or you were thinking about something else?